clr-k8s-examples/README.md

```diff
@@ -34,9 +34,9 @@ This script ensures the following
 * Customizes the system to ensure correct defaults are setup (IP Forwarding, Swap off,...)
 * Ensures all the dependencies are loaded on boot (kernel modules)
 
-> NOTE: This step is done automatically if using vagrant. When running this script, by default
-the RUNNER is set to use crio, if need to be changed make sure to specify the RUNNER to be 
-either crio/containerd. 
+> NOTE: This step is done automatically if using vagrant. The setup_system.sh script uses
+the runtime specified in the RUNNER environment variable and defaults to crio. To use the
+containerd runtime, set the RUNNER environment variable to containerd.
```

Comment:
Nit : Please use the following
The setup_system.sh script uses the runtime specified in the `RUNNER` environment variable and defaults to `crio`. To use the containerd runtime, set the RUNNER environment variable to `containerd`.

Reply:
Reference: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#code-and-syntax-highlighting